x-pack/plugins/apm/server/lib/helpers/create_es_client/call_async_with_debug.ts

```diff
@@ -39,20 +41,24 @@ export async function callAsyncWithDebug<T>({
     return cb();
   }
 
-  const startTime = process.hrtime();
+  const hrStartTime = process.hrtime();
+  const startTime = Date.now();
 
   let res: any;
-  let esError = null;
+  let esError: WrappedElasticsearchClientError | null = null;
+  let esRequestStatus: RequestStatus = RequestStatus.PENDING;
   try {
     res = await cb();
+    esRequestStatus = RequestStatus.OK;
   } catch (e) {
     // catch error and throw after outputting debug info
     esError = e;
+    esRequestStatus = RequestStatus.ERROR;
   }
 
   if (debug) {
     const highlightColor = esError ? 'bgRed' : 'inverse';
-    const diff = process.hrtime(startTime);
+    const diff = process.hrtime(hrStartTime);
```

Comment:
I'm wondering if this is even useful? The elasticsearch response returns its own timer `"took" : 1038`. Perhaps we should just use that and not worry about the network latency.

Reply:
You're probably right, but I'll leave this as-is for now.